executor/adapter.go

```diff
@@ -81,13 +83,18 @@ func (a *statement) IsDDL() bool {
 	return a.isDDL
 }
 
+// Exec implements the ast.Statement Exec interface.
+// This function builds an Executor from a plan, if the Executor doesn't return result,
+// like INSERT, UPDATE statements, it executes in this function, if the Executor returns
```

Comment:
->like the INSERT, UPDATE statements, it executes in this function. If the Executor returns